ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/json/RaoParametersDeserializer.java

```diff
@@ -42,6 +42,10 @@ public RaoParameters deserialize(JsonParser parser, DeserializationContext deser
             switch (parser.getCurrentName()) {
                 case VERSION:
                     parser.nextToken();
+                    String version = parser.getValueAsString();
+                    if (!RAO_PARAMETERS_VERSION.equals(version)) {
```

Comment:
I don't remember this decision.. but I think it's not worth the hassle to support old versions. do you prefer I bump the version  to 3.0?

Reply:
Is there no way to make 2.0 versions deserializable?

Reply:
There is, but it means that we handle backwards compatibility for RaoParameters, which has not been the case. It would require extra code and tests, growing over time, like with CRAC